### PR TITLE
docs(TryItOut): increse documentation coverage

### PR DESCRIPTION
**Pull request by char0n**
### Description
Increase documentation coverage for TryItOut functionality

### Motivation and Context

Multiple requests for better documentation

### How Has This Been Tested?

All the code examples used inside the docs have been tested.

### Types of changes

- [x] No code changes (changes to documentation, CI, metadata, etc)
- [ ] Dependency changes (any modification to dependencies in `package.json`)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: docs/TRY_IT_OUT_EXECUTOR.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+- `String=""` means a String type with a default value of `""`.
+- `String=["a"*, "b", "c", "d"]` means a String type that can be `a, b, c, or d`, with the `*` indicating that `a` is the default value.
+
+Property | Description
+--- | ---
+`spec` | `Object`, **REQUIRED**. OpenAPI definition represented as [POJO](https://en.wikipedia.org/wiki/Plain_old_Java_object).
+`operationId` | `String`. Unique string used to identify an operation. If not provided, `pathName` + `method` must be used instead.
+`pathName` | `String`. OpenAPI defines a unique operation as a combination of a path and an HTTP method. If `operationId` is not provided, this property must be set.
+`method` | `String=["GET", "HEAD", "POST", "PUT", "DELETE", "CONNECT", "OPTIONS", "TRACE", "PATCH"]`. OpenAPI defines a unique operation as a combination of a path and an HTTP method. If `operationId` is not provided, this property must be set.
+`parameters` | `Object`. Parameters object, eg: `{ q: 'search string' }`. Not recognized parameters (not defined in `spec`) will be ignored.
```
Comment (tim-lai) [inline docs/TRY_IT_OUT_EXECUTOR.md:19]: "unrecognized" and "unsupported" parameters

File: docs/TRY_IT_OUT_EXECUTOR.md [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+`requestContentType` | `String`. Sets [appropriate media type](https://swagger.io/docs/specification/describing-request-body/) for request body, e.g. `application/json`. If supplied media type is not defined for the request body, this property is ignored.
+`responseContentType` | `String`. Expect [appropriate media type](https://swagger.io/docs/specification/describing-responses/) response, e.g. `application/json`. Creates an `Accept` header in `Request` object.
+`http` | `Function=Http`. A function with an interface compatible with [HTTP Client](HTTP_CLIENT.md).
+`userFetch` | `Function=cross-fetch`. Custom **asynchronous** fetch function that accepts two arguments: the `url` and the `Request` object and must return a [Response](https://developer.mozilla.org/en-US/docs/Web/API/Response) object. More info in [HTTP Client](HTTP_CLIENT.md) documentation.
+
+For all later references, we will always use following OpenAPI 3.0.0 definition when referring
+to a `spec`.
+
+```js
```
Comment (tim-lai) [inline docs/TRY_IT_OUT_EXECUTOR.md:29]: I think our preference is to encourage YAML for api definitions, rather than json.

Comment (char0n) [reply docs/TRY_IT_OUT_EXECUTOR.md:29]: Yesp, `yaml` is more compact and readable. Converted to yaml. THanks

File: docs/TRY_IT_OUT_EXECUTOR.md (hunk 1 above)
Comment (char0n) [reply docs/TRY_IT_OUT_EXECUTOR.md:19]: "unsupported" doesn't seem to be appropriate. What I'm trying to express here, is that all `parameters` that are not defined in original `spec` will not be used in request. I'll try to use more descriptive language. thanks

**Pull request merged**